File: FundamentosDeLaProgramacionLuisJoyanes4taEdicion/chapter05/JAVA/BreakSentence.java
/*Libro:Fundamentos de Programación, Algoritmos, estructura de datos y objetos 4ta edición. 
    Capitulo 05.-Flujo de control II: Estructuras repetitivas
    Problema 09.-Validar un numero de identificacion. 
    Pagina 176
*/

public class BreakSentence{    
    public static void main(String[] args){

           System.out.print("Cuenta del 1 - 25:\n\n");
            for(int i=1; i<=25; i++){
               System.out.print(i+" ");

                if(i==5){
                   System.out.print("\n\nNumero: "+i+" es hora de salir.\n");
                    break;
                }
            }
    }
}

File: FundamentosDeLaProgramacionLuisJoyanes4taEdicion/chapter05/JAVASCRIPT/JS/breakSentence.js
/*Libro:Fundamentos de Programación, Algoritmos, estructura de datos y objetos 4ta edición. 
    Capitulo 05.-Flujo de control II: Estructuras repetitivas
    Problema 09.-Validar un numero de identificacion. 
    Pagina 176
*/

function calculate(){
    var varSpan=document.createElement("span");
    varSpan.setAttribute("id", "idSpan");      

    if(document.getElementById("idBtnCalculate").value=="Ejecutar"){  
        
        for(let i=1; i<=25; i++){
            varSpan.innerHTML+=i+" ";
         
             if(i==5){
                varSpan.innerHTML+="<br><b><b>Numero: "+i+" es hora de salir.";
                break;
             }
         }

        document.getElementById("idLblMessage").appendChild(varSpan);    
        document.getElementById("idBtnCalculate").value="Limpiar";
    }else{
        document.getElementById("idSpan").remove();

        document.getElementById("idBtnCalculate").value="Ejecutar";
    }

    
}